build/media-suite_compile.sh

```diff
@@ -1055,6 +1055,13 @@ if { [[ $rav1e = y ]] || enabled librav1e; } &&
         [[ -f $CARGO_HOME/config ]] && rm -f "$CARGO_HOME/config"
         log "install-rav1e-c" "$RUSTUP_HOME/bin/cargo.exe" \
             cinstall --release --prefix "$PWD/install-$bits" --jobs "$cpuCount"
+
+        compiler_builtins=$(ar t "install-$bits/lib/librav1e.a" | grep -xG compiler_builtins-.*compiler_builtins.*-cgu.0.rcgu.o)
```

Comment:
```suggestion
        compiler_builtins=$(ar t "install-$bits/lib/librav1e.a" | grep -xG "compiler_builtins-.*compiler_builtins.*-cgu.0.rcgu.o")
```
Quote to prevent any potential shell globs